tutorials/dataframe/df106_HiggsToFourLeptons.C

```diff
@@ -45,7 +46,7 @@ using namespace ROOT::RDF::Experimental;
 
 // Define functions needed in the analysis
 // Select events for the analysis
-bool GoodElectronsAndMuons(const ROOT::RVecI &type, const RVecF &pt, const RVecF &eta, const RVecF &phi, const RVecF &e,
+bool GoodElectronsAndMuons(const ROOT::RVecI &type, const RVecF &pt, const RVecF &eta, const RVecF &phi, const RVecF &e, \
```

Comment:
Why the trailing `\`?

Reply:
Missed remains from the macro, I'll remove it, thanks